networkx/algorithms/__init__.py

```diff
@@ -11,6 +11,7 @@
 from networkx.algorithms.cycles import *
 from networkx.algorithms.cuts import *
 from networkx.algorithms.dag import *
+from networkx.algorithms.disjoint_paths import *
```

Comment:
Thanks for your comments, I have updated the header. The docstring and docs is something I will fix once the code is finished. I think it should be imported like this, in order to move the function names in disjoint_paths to the top of the namespace (such as nx.function_name()) instead of having to write out the complete module path. Furthermore, it looks identical to how most other modules (such as shortest_paths and assortativity) are imported. Is there perhaps some additional settings for the test suites I miss to include a new module? If I remove this line the function gets lost from the namespace and nosetests fails.